src/components/useraccount/orderdetail/LineItems.vue

```diff
@@ -0,0 +1,159 @@
+        <div class="col-sm-1 col-sm-offset-2 col-xs-12 text-center quantity-counter">
+          <span class="visible-xs">{{ $t('quantity') }}</span>
+          <span class="quantity-number">{{item.quantity}}</span>
+        </div>
+
+        <div class="col-sm-2 col-xs-7">
+          <div class="text-right cart-item-price">
+            <span class="visible-xs xs-price-title">{{ $t('price') }}</span>
+            <BasePrice :price="item.price"/>
+          </div>
+        </div>
+        <div class="col-sm-2 col-xs-5">
+          <div class="text-right cart-item-price">
+            <span class="visible-xs xs-price-title">{{ $t('total') }}</span>
+            <BaseMoney :money="item.totalPrice"/>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import gql from 'graphql-tag';
+import BaseMoney from '../../common/BaseMoney.vue';
+import BasePrice from '../../common/BasePrice.vue';
+import DisplayableMoneyFragment from '@/components/DisplayableMoney.gql';
+
+export default {
+  components: { BaseMoney, BasePrice },
+
+  data: () => ({
+    me: null,
+  }),
+
+  computed: {
+    orderNumber() {
+      return this.$route.params.orderNumber;
+    },
+  },
+
+  apollo: {
+    me: {
+      query: gql`
+        query orderByOrderNumber($locale: Locale!, $orderNumber: String) {
+          me {
+            order(orderNumber: $orderNumber) {
+              lineItems {
+                id
+                quantity
+                name(locale: $locale)
+                productSlug(locale: $locale)
+                price{
+                    value {
+                      ...DisplayableMoney
+                    }
+                    discounted {
+                      value {
+                        ...DisplayableMoney
+                      }
+                    }
+                  }
+                totalPrice {
+                  ...DisplayableMoney
+                }
+                variant {
+                  images {
+                    url
+                    label
+                  }
+                  sku
+                  attributes {
+                    ... on mainProductType {
+                      colorFreeDefinition {
+                        value(locale: $locale)
+                      }
+                      size {
+                        value
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+        ${DisplayableMoneyFragment}`,
+      variables() {
+        return {
+          locale: this.$i18n.locale,
+          orderNumber: this.orderNumber,
+        };
+      },
+    },
+  },
+};
+</script>
+
+<i18n>
+en:
+  description: "Description"
+  quantity: "Quantity"
+  price: "Price"
+  total: "Total"
+  size: "Size"
+  color: "Color"
+de:
+  description: "Beschreibung"
+  quantity: "Menge"
+  price: "Preis"
+  total: "Gesamtpreis"
+  size: "Größe"
+  color: "Farbe"
+</i18n>
```

Comment:
I can imagine there are a lot of duplication between this code and the cart's code. Follow the DRY principle (Don't Repeat Yourself) and re-use the same code as in cart.